python/paddle/fluid/framework.py

```diff
@@ -224,6 +225,14 @@ def __init__(self,
                         "persistable is {2}. They are not matched".format(
                             self.name, self.persistable, persistable))
 
+        if capacity is not None:
+            if is_new_var:
```

Comment:
Does every new Var need a capacity? Is it channel specific?

Reply:
Yes, `capacity` is specific only to a channel. I just added this information in the docstring comment also.